app/src/main/java/org/p2p/wallet/send/SendFeeRelayerManager.kt

```diff
@@ -94,8 +117,21 @@ class SendFeeRelayerManager(
     fun buildTotalFee(
         sourceToken: Token.Active,
         calculationMode: CalculationMode,
+        tokenExtensions: TokenExtensionsMap,
     ): SendFeeTotal {
         val currentAmount = calculationMode.getCurrentAmount()
+
+        val transferFeePercent: BigDecimal? = tokenExtensions
+            .getTransferFeeConfig()
+            // todo: fix getting epoch, currently it doesn't work!
+            ?.getActualTransferFee(networkObserver.getCurrentEpoch())
```

Comment:
this should be replaced with SolanaRpcRepository.getEpochInfo().epoch, but it also needs to come up with how to call that suspended function or pass through current epoch using arguments instead